packages/byods/src/index.ts

```diff
@@ -0,0 +1 @@
+export {listDataSources, getDataSources} from './apiClient';
```

Comment:
I'm a little confused, why do we have an api.ts and index.ts? And are they supposed to be identical?

Reply:
The api.ts was for typedoc generation. I've removed it now and can be worked on during the typedoc setup